$NetBSD: patch-ab,v 1.4 2008/08/17 08:40:50 dholland Exp $

We don't need the dependencies for building the package once.
Additionally, not every compiler understands the -MG option.

Also, add -fPIC to CFLAGS. It is apparently not (always?) used from
CPPFLAGS and that breaks the build.

--- rules.mk.in.orig	2005-07-02 20:13:51.000000000 -0400
+++ rules.mk.in	2008-08-17 03:30:01.000000000 -0400
@@ -73,7 +73,7 @@ PYTHON     = @PYTHON@
 PERL	   = @PERL@
 RUBY       = @RUBY@
 
-CFLAGS     = @CFLAGS@ -Wall -I$(NEOTONIC_ROOT) @CPPFLAGS@
+CFLAGS     = @CFLAGS@ -Wall -I$(NEOTONIC_ROOT) -fPIC @CPPFLAGS@
 CPPFLAGS   = -I$(NEOTONIC_ROOT) -fPIC @CPPFLAGS@
 OUTPUT_OPTION = -o $@
 LD         = $(CC) -o
@@ -141,9 +141,6 @@ Makefile.depends: $(NEOTONIC_ROOT)/rules
 	@echo "** OSNAME: $(OSTYPE)"
 	@rm -f Makefile.depends
 	@touch Makefile.depends
-	@for II in `find . -maxdepth 1 -name "*.c" -print`; do \
-		gcc -M -MG ${CFLAGS} $$II >> Makefile.depends; \
-	done;
 	@echo "** (done) "
 
 DEPEND_FILE := $(shell find . -name Makefile.depends -print)
